CHANGELOG.md

```diff
@@ -95,6 +95,10 @@
 
 ### Mimir Continuous Test
 
+### Query-tee
+
+* [ENHANCEMENT] Added HTTP over gRPC support to `query-tee` to allow testing gRPC requests to Mimir instances. #1822
+
```

Comment:
Could you rebase `main` and move this up under the `## Grafana Mimir - main / unreleased` section, please? We've cut the Mimir 2.3.0 CHANGELOG in the meanwhile.